File: yourturn_client/lib/utility/ticketnumber_converter.dart
class TicketNumberConverter {
  String fromInt(int value) {
    return String.fromCharCode((value / 100 + 65).toInt()) +
        (value % 100 < 10 ? '0' : '') +
        (value % 100).toString();
  }

  int fromString(String value) =>
      ((value.codeUnitAt(0) - 65) * 100) + int.parse(value.substring(1));
}

File: model/ticketnumber_converter.js
function fromInt(value) {
    return String.fromCharCode(parseInt(value / 100 + 65)) +
        (value % 100 < 10 ? '0' : '') +
        (value % 100).toString();
}

function fromString(value) {
    return ((value.toString().codeCodeAt(0) - 65) * 100) + parseInt(value.substring(1));
}

module.exports = {
    fromInt,
    fromString
}